datamade/la-metro-councilmatic
# Person form rebase

## Overview

Rebases the person-update-form branch and fixes some issues that arose from the rebase. This includes reinserting S3 keys into the encrypted deployment settings files as well as the update forms for headshots and bios, and adding back in urllib as a requirement.

Changes some settings in docker-compose.yml and adds some lines in views to prevent whitespace forms from being submitted.

Displays proper errors for each form.

Closes #909 
Built off of pr #921 

## Testing Instructions

 * Navigate to the board of directors page and select a person
 * The forms should not be visible until logging in
 * Log in via the admin or otherwise, and navigate back to a person
 * Choose a file to submit through the headshot form
   * If the file was valid, the headshot should change
   * Else, an error is displayed
 - Enter text into the bio form and submit
   * Bio should change if non whitespace content was submitted
   * Else, an error is displayed
